.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+default_install_hook_types: [pre-commit, post-checkout, pre-push]
+repos:
+  -   repo: local
+      hooks:
+        -   id: run lintfix
+            name: run lintfix
+            entry: ./pre-commit/pre-commit-scripts/pre-commit.sh
+            pass_filenames: false
+            language: system
+            stages:
+              - commit
+        -   id: clean and build
+            name: clean and build
+            entry: ./pre-commit/pre-commit-scripts/post-checkout.sh
+            pass_filenames: false
+            language: system
+            always_run: true
+            stages:
+              - post-checkout
+        -   id: run tests
+            name: run tests
+            entry: ./pre-commit/pre-commit-scripts/pre-push.sh
+            pass_filenames: false
```

Comment:
should this have `always_run: true` as well? Its not trigerring on my end without it.

Reply:
The `verbose: true` flag might be helpful here too so we can ensure the output of the tests are correct.

Reply:
I think no harm adding always_run: true. We can add that in. But it was working on my end, in fact this hook ran when I pushed the latest commit hahahha.

Reply:
We can enable verbose: true too.

Reply:
> I think no harm adding always_run: true. We can add that in. But it was working on my end, in fact this hook ran when I pushed the latest commit hahahha.

Yes, sorry - correction.

> Its not trigerring on my end without it **sometimes**.

I'm not entirely sure what's the cause, but agreed having `always_run: true` would be no harm.

Reply:
🤔🤔🤔 Please chime back in if you happen to find out what that option does in the context of push! @EltonGohJH